### ISSUE TITLE
The Wifi Pineapple is still booting

### ISSUE THREAD

**Issue by Maedness** [SecurityAddicted/pineapple-ar150#21]
PLEASE, somebody help me. 

i flashed the AR150 with the newest firmware (2.1.2) from the wifi Pineapple page (https://www.wifipineapple.com/downloads)

i flashed it, upgraded it, it restartet and i typed in the new ip (http://172.16.42.1:1471)

and i'm stuck in a loop of "The Wifi pineapple is still booting, Please wait until the WiFi Pineapple has fully booted. The setup will continue automatically."

doesn't matter how long i wait, i won't change, can someone help me please? (: 
![pineapple](https://user-images.githubusercontent.com/40185761/42129384-8274af7a-7cc3-11e8-8a49-7dedeabac1b6.PNG)

**Comment by pddevins** [comment]
ssh in and issue "jffs2reset -y && reboot" seemed to work for me with the same issue on an earlier version of the firmware.

**Comment by racompton** [comment]
I'm having this issue as well.  What is the user/pass that I should use to log in?

**Comment by pddevins** [comment]
I know this is the worst answer possible, but I don’t remember. It must have been something obvious like ‘root’/‘toor’ or ‘root’/‘password’ or something like that because I did get in and issue that command. (I specifically remember because the reboot command didn’t end up working and I had to do ‘shutdown now’.) My advice is just start trying a bunch of standard, default root passwords.

**Comment by racompton** [comment]
Thanks for the reply.  I am running hydra now to brute force the root password.  I'm using the rockyou.txt word list which will ONLY take 30 days!

**Comment by MR5A1NT** [comment]
You should be able to ssh in with no username or password.

**Comment by racompton** [comment]
Can you be more specific on how I would ssh in with no username?  I didn't think that was possible.  I thought that you had to ssh in with a username.

**Comment by grafi** [comment]
I had the same problem and "jffs2 reset -y && reboot" worked. I successfully ssh'ed using "root" and the wifi/admin password I had on the AR150 before flashing.

**Comment by arifsbc** [comment]
the root password are in the back so it's goodlife

**Comment by d1slact0r** [comment]
new version:
https://github.com/d1slact0r/pineapple-firmware-builder

**Comment by rohitereddy** [comment]
hi, 
i have installed firmware version 2.0 , and im stuck at the wifi pineapple still booting page 

pls help

**Comment by ghost** [comment]
some solution discovered I can't access ssh

**Comment by Oceanlea** [comment]
> some solution discovered I can't access ssh

Same. Tried ssh with no credentials & with creds.  Nothing. Connection refused. 

PLEASE -- can anyone help all of us with this problem?

**Comment by jimi009** [comment]
Got same problem setup stuck after I push button for 2 secs for WiFi, can't ssh in it's driving me nuts ready to bin it.reset it to many times to count . New out the box and not even had chance to use it. Doing a reset when stuck it just dies , nothing .

**Comment by Oceanlea** [comment]
>  it's driving me nuts ready to bin it

Yup - @jimi009, I finally tossed my pineapple in the garbage after a few months of frustration. (That was summer 2020). Sorry to see Hak5 is still pushing out the same sh*t, since 2018 when this thread started.

**Comment by jimi009** [comment]
@Oceanlea yes im getting there. There's nothing online and  their support is well..

**Comment by jimi009** [comment]
@Oceanlea yes im getting there. There's nothing online and  their support is well..

**Comment by jimi009** [comment]
@Oceanlea yes im getting there. There's nothing online and  their support is well..

**Comment by jimi009** [comment]
@Oceanlea got out of loop after last post, I took 12v jack out for 10 secs and put back in, once I got blue light come on I just held reset for 10 secs and couple secs later I opened new browser window and it let me in. Thank god you thought it was a no hoper. I think reset at different stage must do something.
